packages/terra-framework-docs/src/terra-dev-site/doc/compact-interactive-list/Examples.2/CellContent.jsx

```diff
@@ -0,0 +1,147 @@
+import React, { useState } from 'react';
+import CompactInteractiveList, { alignTypes } from 'terra-compact-interactive-list';
+import {
+  IconFeaturedOff, IconFeatured, IconMultipleResultsNormal, IconMultipleResultsNotNormal, IconMultipleResultsCritical,
+} from 'terra-icon';
+import Button from 'terra-button';
+
+// eslint-disable-next-line no-alert
+const buttonCell = <button type="button" aria-label="Learn more button" onClick={() => alert('Learn more button was clicked')}>Learn more</button>;
+const anchorCell = <a href="https://www.oracle.com/" aria-label="Documentation">Documentation</a>;
+const iconResultsNormal = <IconMultipleResultsNormal a11yLabel="Results normal" height="1.5em" width="1.5em" />;
+const iconResultsNotNormal = <IconMultipleResultsNotNormal a11yLabel="Results not normal" height="1.5em" width="1.5em" />;
+const iconResultsCritical = <IconMultipleResultsCritical a11yLabel="Results critical" height="1.5em" width="1.5em" />;
+
+const updateRows = (rowsToUpdate, columns, selectionDetails) => {
+  const columnIndex = columns.findIndex(column => selectionDetails?.columnId === column.id);
+  return [...rowsToUpdate.map(row => {
+    const newRow = {
+      ...row,
+      cells: [...row.cells.map((cell, index) => {
+        const isSelectedCell = selectionDetails?.rowId === row.id && index === columnIndex;
+        // eslint-disable-next-line no-nested-ternary
+        const isSelected = isSelectedCell ? !cell.isSelected : false;
+        return { ...cell, isSelected };
+      })],
+    };
+    return newRow;
+  })];
+};
+
+const FeaturedIcon = () => {
+  const [isFeatured, setIsFeatured] = useState(false);
+  const onButtonClick = () => setIsFeatured(!isFeatured);
+  return (
+    isFeatured ? <Button variant="utility" text="Featured button" icon={<IconFeatured />} onClick={onButtonClick} /> : <Button variant="utility" text="Featured off button" icon={<IconFeaturedOff />} onClick={onButtonClick} />
```

Comment:
Making the name of the button the action that it performs when activated.
```suggestion
    isFeatured ? <Button variant="utility" text="Unfavorite item" icon={<IconFeatured />} onClick={onButtonClick} /> : <Button variant="utility" text="Favorite item" icon={<IconFeaturedOff />} onClick={onButtonClick} />
```